frontend/src/routes/login.lazy.tsx

```diff
@@ -66,7 +66,7 @@ function Login() {
           </h1>
           <div className="card flex flex-col gap-3">
             {data?.errors && (
-              <div style={{ color: "red" }}>
+              <div className="text-red-500 text-sm">
```

Comment:
Errors shouldn't be passed like this in a page or form, but instead use a component because this way the error isn't accessible.